Makefile

```
@@ -75,3 +75,7 @@ www: \
 	dist/apple-touch-icon.png \
 	dist/index.html \
 	dist/manifest.webmanifest
+
+.PHONY: serve
+serve: node_modules
+	$(NODE) ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config ./www/webpack.config.dev.js --open --port 8080
```

This line is a big too long. You can break it with a back slash:

```suggestion
	$(NODE) ./node_modules/webpack-dev-server/bin/webpack-dev-server.js \
    --config ./www/webpack.config.dev.js --open --port 8080
```